rest-api/model/participant_summary.py

```diff
@@ -164,6 +164,11 @@ class ParticipantSummary(Base):
   sampleStatus1PXR2 = Column('sample_status_1pxr2', Enum(SampleStatus), default=SampleStatus.UNSET)
   sampleStatus1PXR2Time = Column('sample_status_1pxr2_time', UTCDateTime)
 
+  # Sample fields for Direct Volunteers
+  sampleStatusDV1SAL2 = Column('sample_status_dv_1sal2',
+                               Enum(SampleStatus), default=SampleStatus.UNSET)
+  sampleStatusDV1SAL2Time = Column('sample_status_dv_1sal2_time', UTCDateTime)
+
```

Comment:
should we also add sample_order_status ?

Reply:
Sorry, I had some confusion and our earlier conversations apparently didn't fix that. I'll add them.